gateway/api/v1/serializers.py

```diff
@@ -43,6 +45,30 @@ def validate(self, attrs):
                 "At least one of attributes (entrypoint, image) is required."
             )
 
+        # validate dependencies
+        # allowlist stored in json config file (eventually via configmap)
+        # sample:
+        # allowlist = { "wheel": ["0.44.0", "0.43.2"] }
+        # where the values for each key are allowed versions of dependency
+        deps = json.loads(attrs.get("dependencies", None))
+        with open(settings.GATEWAY_ALLOWLIST_CONFIG, encoding="utf-8", mode="r") as f:
+            allowlist = json.load(f)
```

Comment:
I forgot this but should we add a try/catch here too? Just in case the file doesn't exist or the json is malformed?

Reply:
does this cover what you were thinking? https://github.com/Qiskit/qiskit-serverless/pull/1443/commits/dfc06056e27f30aa6f210608066d46803a5917d4

Reply:
Yep, I think so. Can we add a `log` warning us about this error? I assume we should always be able to load this file just in case.